src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheckTest.java

```diff
@@ -301,10 +301,23 @@ public void testFinalLocalVariableSwitchExpressions() throws Exception {
             "50:19: " + getCheckMessage(MSG_KEY, "e"),
             "88:19: " + getCheckMessage(MSG_KEY, "e"),
             "122:19: " + getCheckMessage(MSG_KEY, "e"),
-            };
+        };
```

Comment:
@anhminhtran235 please make this as separate commit, such as `minor: fix indentation` since this was not related to #8934. Some GSOC student must've missed this over the summer :)

Reply:
So should it be in this same PR, or in a different PR?

Reply:
> So should it be in this same PR, or in a different PR?

In this PR, please. It is ok for a `minor` commit such as this that doesn't affect functionality.  Thanks for double checking, you are right, it is always a good practice to keep functional changes in separate issues/PRs.

Reply:
I see. I just added a minor commit to fix indentation

Reply:
@anhminhtran235 I forgot:  to pass Travis you will likely need to make it `minor: fix indentation(#8934)`, since one of our CI jobs tests for an issue number in your commits.  Thanks a lot.

Reply:
Thanks @nmancus1. Commit name fixed.

Reply:
Resolved.